combiningHtmlJstDOMLab.js

```diff
@@ -0,0 +1,48 @@
+/*#Question One
+
+a.Make a website that prompts the user
+for a string.Then show a popup that displays the string.*/
+document.addEventListener("DOMContentLoaded", () => {
+    let strOne = prompt("What is your name?")
+    let alertOne = (alert(strOne))
```

Comment:
alert is a function that displays value on the webpage.

A simple `alert(strOne)` would have been enough.